pyproject.toml

```diff
@@ -40,28 +34,10 @@ skip-magic-trailing-comma = true
 [tool.pylint.MASTER]
 fail-on = "I"
 
-[tool.pylint."MESSAGE CONTROL"]
-disable = [
-    "missing-function-docstring",
-    "missing-module-docstring",
-    "missing-class-docstring",
-    "invalid-name",                              # https://github.com/PyCQA/pylint/issues/3401
-    "fixme",
-    "useless-import-alias",                      # required when using no-implicit-reexport in mypy
-    "no-member",                                 # handled by mypy
-    "inherit-non-class",                         # handled by mypy, false positive on ReifiedGeneric
-    "too-few-public-methods",                    # didn't ask + ratio
-    "unused-import",                             # false negatives + handled by flake8
-    "duplicate-code",                            # https://github.com/PyCQA/pylint/issues/214
-    "isinstance-second-argument-not-valid-type", # false positive on Never type, handled by mypy
-    "no-value-for-parameter",                    # handled by mypy
-    "too-many-return-statements",
-]
-enable = ["useless-suppression", "deprecated-pragma"]
-
-[tool.pylint.FORMAT]
-max-line-length = 200
-
 [tool.pytest.ini_options]
 addopts = "--import-mode=importlib"
 xfail_strict = true
+
+[tool.ruff]
+extend-select = ["I", "UP"]
+ignore = ["E501"]
```

Comment:
maybe copy the ruff config from https://github.com/DetachHead/pytest-robotframework/blob/master/pyproject.toml